executor/batch_point_get.go

```diff
@@ -145,6 +149,16 @@ func (e *BatchPointGetExec) Open(context.Context) error {
 	return nil
 }
 
+// Global temporary table would always be empty, so get the snapshot data of it is meanless.
+// globalTemporaryTableSnapshot inherits kv.Snapshot and override the BatchGet methods to return empty.
```

Comment:
Why not skip kv request just like other operators (such as point get)? It's very strange to do it in a different way.

Reply:
Just because the code is different.
There is no such single line of change in BatchPointGet.